inst/qml/UnidimensionalReliabilityBayesian.qml

```diff
@@ -397,7 +393,7 @@ Form
 			}
 			DoubleField
 			{
-				name:			"iwDf"
+				name:			"inverseWishartPriorDf"
 				label:			qsTr("Inverse Wishart df")
 				defaultValue:	vars.count
```

Comment:
Here the upgrader does not work; the df is reset to the default value. However, the Upgrader syntax looks correct. @boutinb @JorisGoosen any idea what could be the cause?

Reply:
```		
ChangeRename
{
		from:	"iwDf"
		to:		"inverseWishartPriorDf"
}
```

does look good yeah

But because `defaultValue: vars.count` I suppose the update gets triggered whenever the count of vars changes (which would most likely happen during loading a qml form)

However I am not so sure about `defaultValue` behaviour and monsieur Bruno would know more I think